InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -14612,7 +14618,8 @@ private string GetNotecardSegment(SceneObjectPart part, string name, int line, i
             {
                 AsyncCommands.
                 DataserverPlugin.DataserverReply(reqIdentifier, NotecardCache.GetLine(assetID, line, startOffset, maxLength));
-                ScriptSleep(DELAY);
+                if (((line % LINES_PER_DELAY) == 0) && (startOffset == 0))
```

Comment:
That isn't the purpose, and wasn't even one of the goals.  They were:
- reduce the time it takes to read a notecard
- do not completely eliminate the penalty because we occasionally have a buggy script that would then spin in a tight loop